src/pybind/mgr/dashboard/frontend/src/app/core/navigation/identity/identity.component.html

```diff
@@ -19,6 +19,15 @@
       <strong>{{ username }}</strong></a>
     </li>
     <li class="dropdown-divider"></li>
+    <li role="menuitem"
+    <li *ngIf="!sso"
+        role="menuitem">
+      <a class="dropdown-item"
+         routerLink="/user-management/users/edit">
+        <i [ngClass]="[icons.lock, icons.width]"></i>
```

Comment:
I have extended all `fa` icons with the `fa-fw`, so we don't need to manually add that anymore.
 ```suggestion
        <i [ngClass]="[icons.lock]"></i>
```